docs/docs/glossary/jamstack.md

```diff
@@ -0,0 +1,32 @@
+---
+title: JAMStack
+disableTableOfContents: true
+---
+
+Learn how to use Gatsby to build websites powered by the JAMStack, a modern architecture that uses JavaScript, APIs and markup instead of a databases and server-side programming languages.
+
+## What is the JAMStack?
+
+JAMStack is a modern architecture for building websites and applications. The _<abbr>JAM</abbr>_ in JAMStack stands for [JavaScript](/docs/glossary#javascript), [APIs](/docs/glossary#api), and HTML markup.
+
+With more traditional websites, such as those built with WordPress or Drupal, content is stored in a database. There's also a presentation layer made of template files, which are sometimes called _views_.
+
+Template files use a mix of [HTML](/docs/glossary#html), CSS, JavaScript, and template tags. Think of template tags as placeholders for pieces of content, e.g. `{{ title }}` for the title of the page.
+
+A software layer then pulls it all together. This layer retrieves content from the database, replaces tags in the template files with the appropriate chunks of content, and returns it all to the browser. A single page gets regenerated each time the server receives a request for that URL.
+
+In this type of architecture, the [frontend](/docs/glossary#frontend) (what you see in the browser) and [backend](/docs/glossary#backend) (the database and software layer) are _tightly coupled_. Both the content and how it's presented are part of the same code base. Content is only available as HTML and only clients that are capable of parsing HTML (browsers, for example) can use it.
+
+### The advantages of JAMStack
```

Comment:
It feels like the only advantage listed here is the freedom to host static files anywhere. Seems like a great opportunity to talk about performance, developer experience, and security in this section.